UnitTests/GitCommandsTests/GitModuleTest.cs

```diff
@@ -452,5 +453,60 @@ public void AddRemote()
             Assert.AreEqual("Please enter a name.", _gitModule.AddRemote("", path));
             Assert.AreEqual("Please enter a name.", _gitModule.AddRemote(null, path));
         }
+
+        [Test]
+        public void GetSubmodulesLocalPaths()
+        {
+            var moduleTestHelpers = new List<CommonTestUtils.GitModuleTestHelper>();
+            try
+            {
+                const int numModules = 4;
+
+                for (int i = 0; i < numModules; ++i)
+                {
+                    moduleTestHelpers.Add(new CommonTestUtils.GitModuleTestHelper($"repo{i}"));
+                }
+
+                foreach (var helper in moduleTestHelpers)
+                {
+                    // TODO: move to GitModuleTestHelper constructor (can't assume all users have global user.name/email configured)
+                    helper.Module.RunGitCmd(@"config user.name ""author""");
+                    helper.Module.RunGitCmd(@"config user.email ""<author@mail.com>""");
+
+                    // Submodules require at least one commit
+                    helper.Module.RunGitCmd(@"commit --allow-empty -m ""Initial commit""");
+                }
+
+                for (int i = numModules - 1; i > 0; --i)
+                {
+                    var parent = moduleTestHelpers[i - 1];
+                    var child = moduleTestHelpers[i];
+
+                    // Add child as submodule of parent
+                    parent.Module.RunGitCmd(GitCommandHelpers.AddSubmoduleCmd(child.Module.WorkingDir.ToPosixPath(), $"repo{i}", null, true));
+
+                    parent.Module.RunGitCmd(@"commit -am ""Add submodule""");
```

Comment:
Was thinking of moving this to GitModuleTestHelper.StageAndCommit(string message = "Commit all changes") or something. Thoughts?